packages/ses/test/test-compartment-make-scope-proxy.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-underscore-dangle */
+import test from 'ava';
+import '../index.js';
+
+lockdown();
+
+test('compartment main use case exploded', t => {
```

Comment:
Perhaps we can frame a test case in terms of usage with a precompiled module functor.